config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```
@@ -578,6 +610,63 @@ presubmits:
         - name: ndots
           value: "1"
 
+  - name: pull-cert-manager-make-e2e-v1-23
+    context: pull-cert-manager-make-e2e-v1-23
+    always_run: false
+    optional: true
+    max_concurrency: 4
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-1.8
+    annotations:
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: jetstack-cert-manager-presubmits-blocking
+      description: Runs the end-to-end test suite against a Kubernetes v1.23 cluster
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-cloudflare-credentials: "true"
+      preset-retry-flakey-tests: "true"
+      preset-enable-all-feature-gates: "true"
+      preset-ginkgo-skip-default: "true"
+    spec:
+      containers:
+      - image: 1.17.8-alpine@sha256:e2e68a9cdd5da82458652fdac3908a3a270686b38039f2829855398e2e06019d
+        args:
+        - runner
+        - make/ci.sh 1.23
```

suggestion: does this need the makefile path?
```suggestion
        - runner
        - make
        - -f
        - make/Makefile
        - ...
```

---

In https://github.com/cert-manager/cert-manager/pull/4914, I have decided to replace the root Makefile with what used to be `make/Makefile`. This won't break any existing Bazel job: the root Makefile's targets weren't used anywhere.

---

@SgtCoDFish I was wrong, the Bazel Prow jobs actually use the old makefile.

---

I have re-introduced the old targets, and have added deprecation warnings in https://github.com/cert-manager/cert-manager/pull/4914/commits/fc2a4273e09bb86e18d2122840c074126802ed90. It looks a bit like this:

```make
# The targets (verify_deps, verify_chart, verify_upgrade, and cluster) are
# temorary and exist to keep the compatibility with the following Prow jobs:
#
#   pull-cert-manager-chart
#   pull-cert-manager-deps
#   pull-cert-manager-upgrade
#
# These targets should be removed as soon as the four above jobs and scripts are
# updated to use the "make" flow.
.PHONY: verify_deps
verify_deps:
	$(warning "The 'verify_deps' target is deprecated and will be removed soon. This target is not useful anymore with the new make flow.")
	./hack/verify-deps.sh

# requires docker
.PHONY: verify_chart
verify_chart:
	$(warning "The 'verify_chart' target is deprecated and will be removed soon. Please use instead 'verify-chart'.")
	bazel build //deploy/charts/cert-manager
	./hack/verify-chart-version.sh bazel-bin/deploy/charts/cert-manager/cert-manager.tgz

.PHONY: verify_upgrade
verify_upgrade:
	$(warning "The 'verify_upgrade' target is deprecated and will be removed soon. Please use instead 'make e2e-setup-kind && ./hack/verify-upgrade.sh'.")
	./hack/verify-upgrade.sh

.PHONY: cluster
cluster:
	$(warning "The 'cluster' target is deprecated and will be removed soon. Please use instead 'make e2e-setup-kind'.")
	./devel/ci-cluster.sh
```